### DISCUSSION TITLE
bool flags with air commands?

### DISCUSSION THREAD

**Discussion by marcusprice** [air-verse/air#317] [Q&A]
Hello! I'm sure I'm missing something simple here, but I'm trying to use a bool flag and it doesn't seem to be picking it up when I run the app with air. When I run the app with the go command (i.e. `go run main.go -init=true`) it works but when I run the same command with air (`air run main.go -init=true`) it just picks up the default value false.

Here's a sample of the code:
```
init := flag.Bool("init", false, "initialize app")
flag.Parse()
log.Println(init)
```
Does anything appear to be wrong in my air command or code? Thanks in advance for your help :)

**Comment by makiuchi-d** [comment]
According to the [README](https://github.com/cosmtrek/air#runtime-arguments) it should be `air -- -init=true`, but it does not work at all.
You have to specify that to `full_bin` in the `.air.toml`.

If you want to do by command line, try [arelo](https://github.com/makiuchi-d/arelo).
```shell
arelo -p '**/*.go' -- go run main.go -init=true
```
